### PR TITLE
Remove babel options. #34

### PR DESCRIPTION
**Pull request by btipling**
#1. It doesn't seem like stage-0 code exists anywhere in this repository.
#2. Putting the transform instructions for browserify in package.json tells upstream users browserify builds to build this project. If they're using babel6 they can't use this project because the babel options in this package.json are incompatible with babel6.

#34

### CODE REVIEW

File: build/imgur.js [hunk 1]
```diff
@@ -16,6 +16,11 @@
         additionalHeaders: {}
     };
 
+    function foo() {
+        Promise.resolve();
+        console.log('done');
```
Comment (Timgur) [inline build/imgur.js]: do we want this in?

Comment (Polyrhythm) [inline build/imgur.js]: nope

**Comment by Polyrhythm** [comment]
:cat2:

**Comment by Timgur** [comment]
:+1:

**Pull request merged**